src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts

```diff
@@ -210,7 +210,7 @@ export class LanguageDetectionSimpleWorker implements ILanguageDetectionWorker {
 			case 'csv':
 			case 'toml':
 				// Other considerations for negativeConfidenceCorrection that
-				// aren't built in but suported by the model include:
```

Comment:
Just some comment typo fix? Seriously

Reply:
Why not?

Reply:
Why not？

Reply:
I was expecting a typo in actual usable source code.